src/tests/ui/planner/workitem-quickpreview.ts

```diff
@@ -239,14 +240,18 @@ export class WorkItemQuickPreview extends ui.BaseElement {
   }
 
   async updateDescription(description: string, append: boolean = false) {
-    await this.descriptionDiv.clickWhenReady();
+    await this.openDescriptionBox();
     if(!append) {
       await this.descriptionTextarea.clear();
     }
     await this.descriptionTextarea.enterText(description);
     await this.descriptionSaveButton.clickWhenReady();
   }
 
+  async openDescriptionBox(){
```

Comment:
add await this.descriptionDiv.clickWhenReady(); before clicking descriptionEditIcon, it will fail with timeout because edit is visible after click()

Reply:
@Raunak1203 you are talking about the openDescriptionBox() function, right?

Reply:
yes